assets/training/model_management/src/run_model_download.py

```diff
@@ -18,6 +18,20 @@
 NEGATIVE_MODEL_NAME_PATTERN = r"[^a-zA-Z0-9-]"
 logger = get_logger(__name__)
 custom_dimensions.app_name = AppName.DOWNLOAD_MODEL
+llama_dict = {
```

Comment:
nit: could we add dict constant to config too?

Reply:
ack